lib/linux_stats/os/proc_mounts.rb

```diff
@@ -42,23 +49,50 @@ module DataFile
   class Reporter
 
     attr_accessor :blocks_per_kilobyte,
-                  :mounted_partitions
+                  :mounted_partitions,
+                  :container_prefix
 
-    def initialize(data_directory = PROC_DIRECTORY_DEFAULT)
+    def initialize(data_directory = PROC_DIRECTORY_DEFAULT, container_mount_name =
```

Comment:
(just a style point)  if there is more stuff than can fit on a line, we normally put one item per line.  e.g.

```
(data_directory = PROC_DIRECTORY_DEFAULT,
 container_mount_name = CONTAINER_MOUNT_PREFIX,
 data = nil,
 test_mode = false)
```